lms/envs/aws.py

```diff
@@ -306,14 +307,47 @@
 ############# CORS headers for cross-domain requests #################
 
 if FEATURES.get('ENABLE_CORS_HEADERS'):
-    INSTALLED_APPS += ('corsheaders', 'cors_csrf')
-    MIDDLEWARE_CLASSES = (
-        'corsheaders.middleware.CorsMiddleware',
-        'cors_csrf.middleware.CorsCSRFMiddleware',
-    ) + MIDDLEWARE_CLASSES
```

Comment:
Note to reviewers: I added an additional feature flag to control the cross-domain cookie, which also depends on `corsheaders` middleware being installed.  I felt it was cleaner to always install the middleware, which can then raise `MiddlewareNotEnabled` to disable itself if the feature flag isn't set.